pom.xml

```diff
@@ -21,7 +21,7 @@
 
   <groupId>io.cdap.plugin</groupId>
   <artifactId>ftp-plugins</artifactId>
-  <version>1.0.0-SNAPSHOT</version>
```

Comment:
pom changes seem to be unnecessary. can we revert them?

Reply:
My maven complains not being able to find cdap 6.4.0 and hydrator 2.6.0. Also I believe current ftp version is 3.0.0

Reply:
why is the version bumped to 3.0.0-SNAPSHOT?

Reply:
Generally we should depend on released versions otherwise you'll see issues like this - 6.4.0-SNAPSHOT not found. So cdap dependency version can be set to 6.4.0

Reply:
Also I think this pr is going to the main. Should it go to main or develop?

Reply:
It seems that there is a mess of branches in this repo. We should validate that develop is superset of main and release/3.0, drop main, make develop default and ensure develop version is >3.0

Reply:
OK, I switched it to develop and dropped main. It looks good now, 3.0 branch do not have any extra changes